controller/handler/dashboard.js

```diff
@@ -15,41 +15,83 @@ const database = require('../../model');
  * @returns {View} Rendered page
  */
 function dashboardView (request, reply) {
-    const trial = database.sequelize.model('trial');
+    const todayDate = moment();
```

Comment:
also use of `moment()` here can be replaced with `new Date()`
which uses one less import :grinning: